lib/internal/Magento/Framework/File/Name.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\File;
+
+use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\Exception\FileSystemException;
+use Magento\Framework\Validation\ValidationException;
+
+/**
+ * Class Name
+ *
+ * @package Magento\Framework\File
```

Comment:
Please, remove the `Class` and `@package` parts of the DocBlock since they are redundant. 
It would be awesome if you could provide a few words about the general purpose of this class in the DocBlock instead. 

Thank you!